__tests__/index.test.js

```diff
@@ -0,0 +1,39 @@
+const app = require('../server/index.js')
+const request = require('supertest')
+const seed = require('../db/seeds/seed.js')
+const data = require('../db/data/test-data')
+const db = require('../db/connection.js')
+
+beforeEach(() => {
+  return seed(data)
+})
+
+afterAll(() => {
+  db.end()
```

Comment:
You may need to return this as its on a new line, if you get messages about not ending gracefully double check.